File: count_braces.js
const fs = require('fs');
const content = fs.readFileSync('c:\\Users\\Administrator\\OneDrive\\Desktop\\pro\\src\\pages\\admin\\Events.jsx', 'utf8');
const openBraces = (content.match(/{/g) || []).length;
const closeBraces = (content.match(/}/g) || []).length;
console.log(`Open: ${openBraces}, Close: ${closeBraces}`);

File: count_braces.py
with open(r'c:\Users\Administrator\OneDrive\Desktop\pro\src\pages\admin\Events.jsx', 'r', encoding='utf-8') as f:
    content = f.read()
    print(f"Open: {content.count('{')}")
    print(f"Close: {content.count('}')}")
